include/votca/tools/linalg.h

```diff
@@ -17,7 +17,7 @@
 
 #ifndef __VOTCA_TOOLS_LINALG_H
 #define	__VOTCA_TOOLS_LINALG_H
-
+#include <votca/tools/votca_config.h>
```

Comment:
I guess here is the point where you would include the different matrix product header and not in config.h!